patches/tModLoader/Terraria/ID/ItemID.TML.cs

```diff
@@ -6,6 +6,17 @@ partial class ItemID
 {
 	partial class Sets
 	{
+		/// <summary>
+		/// The projectile ID of a sandgun ammo item ID. Defaults to 42, as this is the normal sandgun sand projectile.
+		/// This shouldn't be your falling sand projectile - you need to create a second projectile for the sandgun.
+		/// </summary>
+		public static int[] SandgunAmmoToProjectile = Factory.CreateIntSet(42, EbonsandBlock, ProjectileID.EbonsandBallGun, PearlsandBlock, ProjectileID.PearlSandBallGun, CrimsandBlock, ProjectileID.CrimsandBallGun);
+
+		/// <summary>
+		/// The damage a specific sandgun ammo should increase by. Defaults to 0.
+		/// </summary>
+		public static int[] SandgunAmmoDamage = Factory.CreateIntSet(0, EbonsandBlock, 5, PearlsandBlock, 5, CrimsandBlock, 5);
```

Comment:
Should this be titled `SandgunAmmoBonusDamage` instead? Something to communicate its additive properties.